### DISCUSSION TITLE
NGC-Sky Vector from Lumicon supported ?

### DISCUSSION THREAD

**Discussion by usjeff** [Stellarium/stellarium#2598] [Q&A] [answered]
Hi All, 
 I have an NGC-Sky Vector DSC from Lumicon on my Televue Gibraltar alt/az mount. The telescope control tab in Stellarium shows "connected" but the cross hairs don't follow the scope movement. Stellarium is installed on an HP laptop running CentOS 8 linux. The USB trace shows there is some traffic on the interface but the Stellarium debug.log does NOT show anything happening. The NGC-Sky Vector is/was the same as Argo-Navis DSC back in the day. Does anyone know if this old DSC is supported ? It worked fine with "The Sky" years ago.
Thank you for your time.
JeffD

**Comment by gzotti** [comment]
I don't know more than the User Guide. Stellarium supports Meade LX-200 and Celestron NexStar protocols with some variants, and on Linux should support instruments that are supported by INDI or even RTS2. Try to see whether this instrument is supported by INDI.
Stellarium is no derivate of The Sky, so the device support there is no indication for us.

**Comment by usjeff** [comment, marked as answer]
I was able to get the Limicon Sky Vector working on a Windows10 computer by installing ASCOM with the DSC driver. The full instructions are at the website eksfiles.net so I won't repeat them here. The telescope slew dialog now directs you to the selected item. Unfortunately the telescope finder doesn't move to the new item. The INDI driver on Debian Linux did not work at all; it wouldn't even connect.
